### PR TITLE
Move GC and Vulnerability to Tasks

### PR DESCRIPTION
**Pull request by pureshine**
fix pr #6349
Move GC and Vulnerability to Tasks
image

Signed-off-by: FangyuanCheng <fangyuanc@vmware.com>

### CODE REVIEW

File: tests/resources/Harbor-Pages/Configuration.robot [hunk 1]
```diff
@@ -278,7 +278,7 @@ Delete A Label
 ## Garbage Collection	
 Switch To Garbage Collection
     Sleep  1
-    Click Element  xpath=${configuration_xpath}
+    Click Element  xpath=//clr-vertical-nav-group-children/a[contains(.,'Garbage')]
```
Comment (danfengliu) [inline tests/resources/Harbor-Pages/Configuration.robot]: Do not use hard code for xpath locator, please use variable, and variable should be located in element file.

Comment (danfengliu) [reply tests/resources/Harbor-Pages/Configuration.robot]: Click Element  should be replace by anther keyword with retry function.

File: tests/resources/Harbor-Pages/Vulnerability.robot [hunk 2]
```diff
@@ -13,12 +13,15 @@ Disable Scan Schedule
      Retry Element Click  //cron-selection//button[contains(.,'SAVE')]
 
 Go To Vulnerability Config
-    Retry Element Click  //config//button[contains(.,'Vulnerability')]
+    Retry Element Click  //vulnerability-page//button[contains(.,'Vulnerability')]
```
Comment (danfengliu) [inline tests/resources/Harbor-Pages/Vulnerability.robot]: Do not use hard code for xpath locator, please use variable, and variable should be located in element file.

File: tests/resources/Harbor-Pages/Vulnerability.robot [hunk 3]
```diff
@@ -13,12 +13,15 @@ Disable Scan Schedule
...
 
 Trigger Scan Now
-    Retry Element Click  //config//button[contains(.,'NOW')]
+    Retry Element Click  //vulnerability-page//button[contains(.,'NOW')]
     Sleep  10
 
+Switch To Vulnerability Page
+    Retry Element Click  //clr-vertical-nav-group-children/a[contains(.,'Vulnerability')]
```
Comment (danfengliu) [inline tests/resources/Harbor-Pages/Vulnerability.robot]: Do not use hard code for xpath locator, please use variable, and variable should be located in element file.

File: tests/resources/Harbor-Pages/Vulnerability.robot [hunk 4]
```diff
@@ -13,12 +13,15 @@ Disable Scan Schedule
      Retry Element Click  //cron-selection//button[contains(.,'SAVE')]
 
 Go To Vulnerability Config
-    Retry Element Click  //config//button[contains(.,'Vulnerability')]
+    Retry Element Click  //vulnerability-page//button[contains(.,'Vulnerability')]
 
 Trigger Scan Now
-    Retry Element Click  //config//button[contains(.,'NOW')]
+    Retry Element Click  //vulnerability-page//button[contains(.,'NOW')]
```
Comment (danfengliu) [inline tests/resources/Harbor-Pages/Vulnerability.robot]: Do not use hard code for xpath locator, please use variable, and variable should be located in element file.

File: tests/resources/Harbor-Pages/Configuration.robot [hunk 5]
```diff
@@ -290,3 +290,7 @@ Click GC Now
 View GC Details
     Click Element  xpath=${gc_log_details_xpath}
     Sleep  2
+
+Switch To GC History
+    Retry Element Click  xpath=${gc_log_xpath}
```
Comment (danfengliu) [inline tests/resources/Harbor-Pages/Configuration.robot]: Must add a verification for the result of Switch To GC History.

File: src/portal/lib/src/config/vulnerability/vulnerability-config.component.scss [hunk 6]
```diff
@@ -31,12 +31,22 @@
...
 }
-
-.btn-scan-right button{
-    width: 160px;
-    margin-bottom: 0px;
-    margin-top: 5px;
+.button-group {
+    display: flex;
+    align-items: center;
+    .btn-scan-right button{
```
Comment (jwangyangls) [inline src/portal/lib/src/config/vulnerability/vulnerability-config.component.scss]: modify it into the grammar of SCSS -------.btn-scan-right{ button

**Pull request merged**